docker/jdk9/x86_64/ubuntu/Dockerfile-openj9

```diff
@@ -0,0 +1,39 @@
+# Copyright IBM Corporation 2017.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk_container
+
+MAINTAINER Mark Stoodley <mstoodle@ca.ibm.com>
+
+# Install required OS tools
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    autoconf \
+    bison \
+    libffi-dev \
+    libnuma-dev \
+    g++-4.8 \
+    gcc-4.8 \
+&& rm -rf /var/lib/apt/lists/*
+
+RUN rm /usr/bin/gcc \
+    && rm /usr/bin/g++ \
+    && rm /usr/bin/c++ \
+    && ln -s /usr/bin/gcc-4.8 /usr/bin/gcc \
+    && ln -s /usr/bin/g++-4.8 /usr/bin/g++ \
+    && ln -s /usr/bin/g++-4.8 /usr/bin/c++
+
+ARG OPENJDK_VERSION
+ENV OPENJDK_VERSION=$OPENJDK_VERSION
+ENV JDK_PATH=jdk
```

Comment:
I like how you've done this. @EnriqueL8 just added this [PR](https://github.com/AdoptOpenJDK/openjdk-build/pull/164) to do something similar but instead of passing to the docker file, we can ofcourse hard code it. Do you want to move `ENV JDK_PATH=jdk` to the parent dockerfile? We should also add `ENV JDK_PATH=j2sdk-image` to the jdk8u dockerfile

Reply:
Sure, I can do that.